src/scss/_base.scss

```diff
@@ -19,8 +19,8 @@ body {
   overflow-x: hidden;
 }
 
-img:not(.dark-bg *, .footer *, .blog-posts *),
-svg:not(.dark-bg *, .footer *, .blog-posts *) {
+img:not(.nav *, .footer *, .blog-posts *),
```

Comment:
If you change/remove the `<nav>` class name don't forget the corresponding CSS :)